File: Beginner/javascript/b1013.js
var input = require("fs").readFileSync("stdin", "utf8");

var values = input.split(" ");

var a = parseInt(values.shift());
var b = parseInt(values.shift());
var c = parseInt(values.shift());

var maiorAB = ((a + b + Math.abs(a - b))/2);
var maiorABC = ((maiorAB + c + Math.abs(maiorAB - c))/2);

console.log(maiorABC + " eh o maior");

File: python/b1013.py
a, b, c = map(int, input().split(" "));
maiorAB = (a + b + abs(a - b)) / 2;
maiorABC = (maiorAB + c + abs(maiorAB - c)) / 2;

print(f"{maiorABC:.0f} eh o maior");